### ISSUE TITLE
Add support for hyperlinks

### ISSUE THREAD

**Issue by ianchow** [paintparty/bling#10]
Would be nice if there's a tag for hyperlinks eg
`[:a {:href "example.com"} "this is a link"]`
ref: https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda

**Comment by paintparty** [comment]
Yes i agree, this has been on my list of things to look at and is in the hopper for the next release.
